website/docs/r/app_service_certificate_order.html.markdown

```diff
@@ -25,7 +25,7 @@ resource "azurerm_app_service_certificate_order" "example" {
   resource_group_name = "${azurerm_resource_group.example.name}"
   location            = "global"
   distinguished_name  = "CN=example.com"
-  product_type        = "standard"
+  product_type        = "Standard"
```

Comment:
was already cased correctly in docs. got this error
```
Error: Error setting `product_type` for App Service Certificate Order "usos1cer01-inmp-dev" (Resource Group "usos1rgp01-certs"), either `Standard` or `WildCard`
```

when using all lower case.